### DISCUSSION TITLE
New pattern for poncho

### DISCUSSION THREAD

**Discussion by darigovresearch** [freesewing/freesewing#1618] [Forum]
**What is it that you would like to see happen?**

Ponchos are great but usually are too large, too small or are one time use as they are made of non-recyclable and flimsy material, having a freesewing pattern means you can resolve all those problems. Idea is to fork the hood code from [huey](https://freesewing.org/designs/huey/), [hugo](https://freesewing.org/designs/hugo/) or [Yuri](https://freesewing.org/designs/yuri/) and add simple variations such as square, circle rectangle for the poncho body as an initial MVP (minimal viable product). Can be made from upcycled material like torn sail, tarp or tent canvas and can be used as further material to make other freesewing garments if you make a mistake or the poncho becomes worn out.

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**

This will likely be one of our first patterns to see what the contribution process is like & how to generate parametric patterns. Anybody who is new to contributing or interested in helping is welcome to join. See this video if you need some guidance on how you may be able to help - https://www.youtube.com/watch?v=UWA4wyacY2A

Suggested name is Paul the poncho but suggestions are welcome

**Comment by darigovresearch** [comment]
Was suggested on the latest contributor call (https://github.com/freesewing/freesewing/discussions/1616) that huey would be the simplest pattern for the hood to adapt as an MVP

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions

**Comment by darigovresearch** [comment]
Reopening discussion from the community call https://github.com/freesewing/freesewing/discussions/5532

**Comment by BenJamesBen** [comment]
Closing because there does not seem to be any interest for active discussion about this topic, per the April 27, 2024 Community Call. https://github.com/freesewing/freesewing/discussions/5724#discussioncomment-9247197

(Please feel welcome to re-open this Discussion if there is interest in resuming active discussion about the topic.)
